src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -279,14 +285,22 @@ export function getMostRecentBundleBlockRanges(
 
     // If UBA is not enabled for this bundle, exit early since no subsequent bundles will be enabled
     // for the UBA, as the UBA was a non-reversible change.
-    const hubPoolStartBlock = getBlockRangeForChain(rootBundleBlockRanges, hubPoolClient.chainId)[0];
+    const hubPoolStartBlock = getBlockRangeForChain(
+      rootBundleBlockRanges,
+      hubPoolClient.chainId,
+      hubPoolClient.configStoreClient.enabledChainIds
+    )[0];
     if (!isUbaBlock(hubPoolStartBlock, hubPoolClient.configStoreClient)) {
```

Comment:
Does this account for if hubPoolStartBlock is undefined?

Reply:
It should never be `undefined` here and thats enforced by the type